schematic/__init__.py

```diff
@@ -0,0 +1 @@
+from .configuration import CONFIG
```

Comment:
Do we want to replace this with `from schematic.configuration import CONFIG`? The way it is right now is good enough, just checking what would be best.

Reply:
Agreed. It's better to be consistent with the rest of the package.